docs/src/enterprise/getstarted/quickstart.md

```diff
@@ -104,26 +104,28 @@ You can choose from the following options:
     ```
 
 === "Advanced (SSO Enabled)"
-    This setup uses OIDC as the SSO authentication method thus requiring a valid OIDC configuration.
+This setup uses OIDC as the SSO authentication method thus requiring a valid OIDC configuration.
```

Comment:
What's going on with all this formatting thats unrelated to your scope? 
You somehow managed to break the tabs and now its 2 different boxes, the SSO disabled and advanced. Look at the UI preview.